### PR TITLE
Release notes for Mendix for Private Cloud - 10 December

### PR DESCRIPTION
**Pull request by zlogic**
Updated documentation and Release Notes for the new version of Mendix Private Cloud. We're expecting to release it on Thursday, 10 December.

### CODE REVIEW

File: content/developerportal/deploy/private-cloud-cluster.md [hunk 1]
```diff
@@ -423,7 +443,62 @@ When choosing the **Existing docker-registry secret**, you will need to add this
...
+       -----BEGIN CERTIFICATE-----
+       [...]
+       -----END CERTIFICATE-----
+       ```
+       (concatenate all the public keys from custom CAs into one `custom.crt` file, separating them with line breaks and optional comments).
+   2. Load into a file (replace `{namespace}` with the namespace where the Operator is installed):
+
+        For OpenShift:
+        ```shell
+        oc -n {namespace} create secret generic mendix-custom-ca --from-file=custom.crt=custom.crt
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-cluster.md:475]: @zlogic
Is `mendix-custom-ca` just an example or is it a literal?
Perhaps we should document it as <custom ca filename>?

File: content/developerportal/deploy/private-cloud-config-script.md [hunk 2]
```diff
@@ -363,3 +382,59 @@ When choosing the **Existing docker-registry secret**, you will need to add this
...
+       -----BEGIN CERTIFICATE-----
+       [...]
+       -----END CERTIFICATE-----
+       ```
+       (concatenate all the public keys from custom CAs into one `custom.crt` file, separating them with line breaks and optional comments).
+   2. Load into a file (replace `{namespace}` with the namespace where the Operator is installed):
+
+        For OpenShift:
+        ```shell
+        oc -n {namespace} create secret generic mendix-custom-ca --from-file=custom.crt=custom.crt
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-config-script.md:418]: Same comment - is mendix-custom-ca a literal or a variable name?

File: content/developerportal/deploy/private-cloud-supported-environments.md [hunk 3]
```diff
@@ -138,11 +138,18 @@ For every Mendix app environment, a new database schema and user (role) will be
...
+By default, Mendix for Private Cloud will first try to connect with TLS enabled; if the server doesn't support TLS, the Mendix Operator will reconnect without TLS.
+To ensure compatibility with all PostgreSQL databases (including ones with self-signed certificates), all TLS CAs are trusted by default.
 
-Mendix for Private Cloud will first try to connect with TLS enabled; if the server doesn't support TLS, the Mendix Operator will retry connecting without TLS.
+If Strict TLS is enabled, Mendix for Private Cloud will connect with TLS and validate the Postgres server's TLS certificate. In this case, the connection will fail if 
 
-To ensure compatibility with all PostgreSQL databases (including ones with self-signed certificates), all TLS CAs are trusted by default.
+* the Postgres server has an invalid certificate
+* or its certificate is signed by an unknown Certificate Authority
+* or doesn't support TLS.
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-supported-environments.md:148]: If what doesn't support TLS, the **connection**?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
@zlogic 
Can you answer these questions - thanks.

File: content/developerportal/deploy/private-cloud-cluster.md (hunk 1 above)
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-cluster.md]: Yes, this is an example - I've changed it to `{secret}` in a new commit.

File: content/developerportal/deploy/private-cloud-config-script.md (hunk 2 above)
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-config-script.md]: Yes, this is an example - I've changed it to `{secret}` in a new commit.

File: content/developerportal/deploy/private-cloud-supported-environments.md (hunk 3 above)
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-supported-environments.md]: I've rephrased it to _or the PostgreSQL server doesn't support TLS connections_ in a new commit

**Review by MarkvanMents — APPROVED** [review]
Great - ready to be published.

**Comment by zlogic** [comment]
Hi @MarkvanMents, can you please publish PR now?

**Pull request merged**